docs/lsm/quickstart/quickstart.rst

```diff
@@ -82,13 +82,11 @@ This folder contains a project.yml, which looks like this:
     copyright: 2022 Inmanta
     modulepath: libs
     downloadpath: libs
-    repo:
-    - type: package
-        url: https://packages.inmanta.com/public/quickstart/python/simple/
-    - type: package
-        url: https://packages.inmanta.com/<token>/inmanta-service-orchestrator-|version_major|-stable/python/simple/
     install_mode: release
-    requires:
+    # This example requires licensed modules,
```

Comment:
The `install_mode` option is deprecated in favor of `pip.pre`, so we should not set it here anymore.

Reply:
I removed it everywhere